src/models/applicationState.ts

```diff
@@ -500,6 +500,7 @@ export enum FieldType {
     Time = "time",
     Integer = "integer",
     SelectionMark = "selectionMark",
+    Image ="image",
```

Comment:
There's a recent requirement change just confirmed that please change the "Image" FieldType to "Signature" FieldType, the other requirements remain the same.